ghub.py

```diff
@@ -332,8 +338,8 @@ def display_pull_requests(verbose=False, number=None):
     if not pullreqs:
         print("No results.")
         return
-    if number:
-        pullreqs = (pullreqs, )
+
+    pullreqs = (pullreqs, )
```

Comment:
I'm not sure there's a bug being fixed here and I don't think this change improves readability, so I'd say let's leave this (and changes at 465-468) as-is.

Reply:
@ccstolley You are indeed correct. The original code actually handles this well. Will address the feedback. Thank you!